packages/elements/source/composite/alert/page.ts

```diff
@@ -1,156 +1,103 @@
 import * as token from '@universityofmaryland/web-styles-library/token';
-import { shrinkThenRemove } from '@universityofmaryland/web-utilities-library/animation';
-import { close as iconClose } from '@universityofmaryland/web-icons-library/controls';
 import { warning as iconWarning } from '@universityofmaryland/web-icons-library/indicators';
-import {
-  CreateAlertText as AlertText,
-  TypeAlertTextProps,
-} from './elements/text';
-
-type TypeShouldShowProps = {
-  daysToHide?: string;
-};
-
-type TypeAlertProps = TypeShouldShowProps &
-  TypeAlertTextProps & {
-    isThemeLight?: boolean;
-    isThemeDark?: boolean;
-    isShowIcon?: boolean;
-  };
-
-type TypeAlertButtonProps = {
-  container: HTMLElement;
+import { CreateAlertText as AlertText, AlertTextProps } from './elements/text';
+import ElementBuilder from '@universityofmaryland/web-builder-library';
+import { type ElementVisual } from '../../_types';
+import { BREAKPOINTS } from './globals';
+import { CreateCloseButton } from './elements/closeButton';
+
+export interface AlertPageProps extends AlertTextProps {
+  isThemeLight?: boolean;
+  isShowIcon?: boolean;
+}
+
+const { MEDIUM } = BREAKPOINTS;
+
+const createIcon = (iconMarkup: string): ElementVisual => {
+  const iconWrapper = ElementBuilder.create.div({
+    className: 'alert-page-icon',
+    elementStyles: {
+      element: {
+        display: 'block',
+        fill: token.color.gold,
+        [`@container (max-width: ${MEDIUM}px)`]: {
```

Comment:
Sooo nitpicky of me but would like space on top of every identifier for readability :DDD